Drivers/BSP/Components/nt35510/nt35510.c

```diff
@@ -100,7 +100,7 @@ uint8_t NT35510_Init(uint32_t ColorCoding, uint32_t orientation)
   const uint8_t nt35510_reg16[] = {0x55, 0xAA, 0x52, 0x08, 0x00, 0xF0};
   const uint8_t nt35510_reg17[] = {0xFC, 0x00, 0xB1};
   const uint8_t nt35510_reg18[] = {0xB6, 0x03};
-  const uint8_t nt35510_reg19[] = {0xB5, 0x51};
+  const uint8_t nt35510_reg19[] = {0xB5, 0x50};
```

Comment:
Value `0xB5` should also be set to `0x50`.

Reply:
Hi @ALABSTM 
The value 0xB5 is the address of the DPRSLCTR (Display Resolution Control) register. If I change this value, specifically to 0x50, the write operation will no longer be performed on the DPRSLCTR register. Additionally, in the datasheet, I couldn't find any reference to a register with the address 0x50.

/**
  * @brief  DCS or Generic short/long write command
  * @param  NbrParams: Number of parameters. It indicates the write command mode:
  *                 If inferior to 2, a long write command is performed else short.
  * @param  pParams: Pointer to parameter values table.
  * @retval HAL status
  */
void DSI_IO_WriteCmd(uint32_t NbrParams, uint8_t *pParams)
{
  if(NbrParams <= 1)
  {
   HAL_DSI_ShortWrite(&hdsi_discovery, LCD_OTM8009A_ID, DSI_DCS_SHORT_PKT_WRITE_P1, pParams[0], pParams[1]); 
  }
  else
  {
   HAL_DSI_LongWrite(&hdsi_discovery,  LCD_OTM8009A_ID, DSI_DCS_LONG_PKT_WRITE, NbrParams, pParams[NbrParams], pParams); 
  } 
}

if NbrParams <= 1 (our case), first you transmit the address and then the value.
So, IMHO opinion, your request is wrong.
Could you kindly explain to me why you made this request?

Thanks and regards,
Dario

Reply:
Hi @passgat,

Thank you for this feedback. Please let me double-check from my side. I'll keep you posted.

With regards,

Reply:
Hi @passgat,

Thank you again for your feedback. You are absolutely right. The first value is the register's address and should be `0xB5`.

Actually, I confused this version of the NT35510 driver and a newer one supported by the **STM32CubeH7** firmware, where registers' addresses are handled differently.

I retrieve my remark and approve your change.

With regards,